modules/apigee_edge_apidocs/apigee_edge_apidocs.info.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Apigee Edge API Docs'
+name: 'Apigee Edge API Docs (deprecated)'
```

Comment:
I would rather add it to a new `Apigee (Deprecated)` group.

Reply:
@cnovak What do you think about this suggestion?

Reply:
I left the module name as is but changed the package to `Apigee (Deprecated)` for good measure.